### PR TITLE
Mention apt install method

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

**Comment by arzzen** [comment]
Thank you for your PR, are you a maintainer of these releases?

I looked at the list of packages and there is only support for the Ubuntu `focal` and `groovy` release https://packages.ubuntu.com/search?keywords=git-quick-stats and for Debian https://pkgs.org/download/git-quick-stats

It would be nice to see the installation of git-quick-statistics for all releases (at least for all LTS and current releases), not just for specific Ubuntu / Debian releases :)

**Comment by SuperSandro2000** [comment]
> are you a maintainer of these releases?

Nope.

It only got recently added to the repos (see https://tracker.debian.org/pkg/git-quick-stats). So it may or may not receive backports in the future.

**Comment by arzzen** [comment]
Maybe @bisco2 can help us ;) (I guess he is a maintainer https://tracker.debian.org/pkg/git-quick-stats)

**Comment by SuperSandro2000** [comment]
But isn't this a separate issue unrelated to this MR?

**Comment by tomice** [comment]
Is there a technical reason that is stopping us from having this package support Xenial and Bionic? Those are still popular releases that I'd like to see supported, and they're actually some of the main ones I use on a day-to-day basis in my field of work.

**Comment by SuperSandro2000** [comment]
> Is there a technical reason that is stopping us from having this package support Xenial and Bionic?

Someone needs to do it. Probably no big ones cause it is only shell.

> Those are still popular releases that I'd like to see supported

Which is a valid argument which has nothing to do with this PR cause right now you are not informed that this project is packaged at all. More Distros will add support over time and we can't wait until every last LTS version has this package.

> and they're actually some of the main ones I use on a day-to-day basis in my field of work.

And I mainly use Debian Testing and Ubuntu latest all day long and almost missed that this is packaged and this PR is delayed for no good reason.

**Comment by tomice** [comment]
As the current PR stands, the "UNIX and Linux" section no longer makes sense once we start adding package managers for Linux. How is "Debian" different from "Linux" and which one should the user choose over the other? It should be broken out into package manager-based installs for the currently supported methods, as well as options for installing via source. We could also look into moving the macOS brew installer into the package manager section, too.

**Comment by bisco2** [comment]
> Maybe @bisco2 can help us ;) (I guess he is a maintainer https://tracker.debian.org/pkg/git-quick-stats)

Yes, I am, at least for the Debian package. I don't know who made the Ubuntu packages and how they are uploaded to different releases. For Debian, I didn't plan to make a backport for `buster`, but if time permits I can do that. I don't see any versioned dependencies, so it should be straightforward. For now it would probably be more helpful to mention the releasename `bullseye` in the apt install method (and later `buster-backports` when the backport was uploaded).

**Comment by arzzen** [comment]
@bisco2 thanks for your reply
@SuperSandro2000 please mention the release name `bullseye` (with distro name)  in the readme.  
Then this PR can be accepted.

File: README.md [hunk 1]
```diff
@@ -190,6 +190,12 @@ export _MENU_THEME=legacy
 
 ## Installation
 
+### Debian and Ubuntu
+
```
Comment (SuperSandro2000) [inline README.md:194]:
```suggestion

If you are on at least Debian Bullseye or Ubuntu Focal you can use apt for installation:

```

Comment (SuperSandro2000) [reply README.md:194]: @arzzen if you want to change the wording feel free to do so.

**Pull request merged**